op-node/p2p/sync.go

```diff
@@ -231,6 +234,7 @@ func NewSyncClient(log log.Logger, cfg *rollup.Config, newStream newStreamFn, rc
 		requests:        make(chan rangeRequest), // blocking
 		peerRequests:    make(chan peerRequest, 128),
 		results:         make(chan syncResult, 128),
+		globalRL:        rate.NewLimiter(globalServerBlocksRateLimit, globalServerBlocksBurst),
```

Comment:
Why would we limit the total number of requests we send? As long as we spread it out across enough peers we don't need to throttle ourselves.  Restricting the number of concurrent requests we have in-flight makes sense (and we do that I believe) but not a global rate limit (unless I'm missing something).

Reply:
Yeah you're right - we have a limit on the number of pending requests in `peerRequests` but only limit to 1 in-flight request per peer.  I'm on the fence about this - what are we trying to protect against with this?
I'm just always nervous about limiting the speed sync can catch up because you generally can't do anything else useful until you're back in sync.

Reply:
We're trying to limit how many blocks are fetched over P2P. It's set to 20 RPS. In general nodes should try to get data from L1 instead.